### PR TITLE
Update datadog/dd-trace-py python versions

### PR DESCRIPTION
**Pull request by brettlangdon**
Updating all versions to the latest
Making 3.9 the default Python version
Adding 3.10-dev

### CODE REVIEW

**Review by Kyle-Verhoog — APPROVED** [review]
:+1: Note that we'll have to wait until this is merged in order for the docker image to be deployed to dockerhub

**Comment by brettlangdon** [comment]
yeah, we'll need a follow-up PR for actually using 3.10

**Review by P403n1x87 — APPROVED** [review]

**Pull request merged**
